packages/web-runtime/src/services/uppyService.ts

```diff
@@ -34,11 +34,30 @@ export class UppyService {
 
   constructor() {
     this.uppy = new Uppy({
-      autoProceed: true
+      autoProceed: true,
+      locale: {
+        strings: {
+          // for some reason this string is required and missing in uppy
+          addedNumFiles: 'Added %{numFiles} file(s)'
```

Comment:
Don't we need to pass this through `$gettext`?

Reply:
We don't actually use the string (which makes the error without even more confusing), but you're right, it would be correct to translate it anyways.